### PR TITLE
Feature/lock color palette

### PR DESCRIPTION
**Pull request by laytonmaes**
# Description
Add lock and unlock functionality, add custom lock icon, add functionality to change lock icon when locking/unlocking colors.

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring

# How Has This Been Tested?
- [x] open localHost
- [x] dev tools

# Checklist:
- [x] My code follows the Turing's guidelines of this project
- [x] I have performed a self-review of my own code
- [x] No console error messages

### CODE REVIEW

**Comment by kal-aalrajhi** [comment]
It looks so good!!

File: index.html [hunk 1]
```diff
@@ -8,43 +9,43 @@
       <h1>ColoRandom</h1>
       <section class="current-palette">
         <div id="color1">
-          <div class="box" id="box1">
+          <div class="box box-1" id="box1">
```
Comment (whiteheadol) [inline index.html:12]: I do want to revisit this naming convention. I'm not sure that two classes with such similar names is what we want!

File: index.html [hunk 2]
```diff
@@ -8,43 +9,43 @@
...
+            <img class="lock" src="./assets/lock-unlocked.svg">
           </div>
         </div>
         <div id="color5">
-          <div class="box" id="box5">
+          <div class="box box-5" id="box5">
           </div>
           <div class="hexcode">
             <p>#123456</p>
-            <img src="./assets/padlock.png">
```
Comment (whiteheadol) [inline index.html:47]: I like that we found a way to do this without toggling 'hidden' on and off for two different images.

File: scripts.js [hunk 3]
```diff
@@ -65,6 +56,20 @@ function displayCurrentPalette() {
   }
 }
 
+function lockColor(event) {
```
Comment (whiteheadol) [inline scripts.js:59]: Nick did give us a hint that led us to: currentPalette[`color${i+1}`].locked = !currentPalette[`color${i+1}`].locked

**Review by kal-aalrajhi — APPROVED** [review]
Looks good!

**Pull request merged**